### PR TITLE
Fixed lint errors

### PR DESCRIPTION
**Pull request by Bhavnaharitsa**
Fixes : Fixed lint errors in Device Connect Fragment , dimens.xml and gradlew

Changes:Inspected code using Android Lint and fixed errors in the above files

### CODE REVIEW

File: app/src/main/java/org/fossasia/susi/ai/device/deviceconnect/DeviceConnectFragment.kt [hunk 1]
```diff
@@ -253,6 +254,7 @@ class DeviceConnectFragment : Fragment(), IDeviceConnectView {
         }
     }
 
+    @SuppressLint("RestrictedApi")
```
Comment (iamareebjamal) [inline app/src/main/java/org/fossasia/susi/ai/device/deviceconnect/DeviceConnectFragment.kt]: This is not fixing

File: gradlew [hunk 2]
```diff
@@ -134,9 +134,9 @@ if $cygwin ; then
         CHECK2=`echo "$arg"|egrep -c "^-"`                                 ### Determine if an option
 
         if [ $CHECK -ne 0 ] && [ $CHECK2 -eq 0 ] ; then                    ### Added a condition
-            eval `echo args$i`=`cygpath --path --ignore --mixed "$arg"`
+            eval `echo args$i"="cygpath --path --ignore --mixed "$arg"`
         else
-            eval `echo args$i`="\"$arg\""
+            eval `echo args$i"="\"$arg\""
```
Comment (iamareebjamal) [inline gradlew]: Unneeded changes

File: app/src/main/res/values/dimens.xml [hunk 3]
```diff
@@ -122,6 +122,7 @@
     <dimen name="arrow_width">16dp</dimen>
     <dimen name="arrow_height">16dp</dimen>
     <dimen name="arrow_padding_large">56dp</dimen>
+    <dimen name="activity_horizontal_margin">64dp</dimen>
```
Comment (iamareebjamal) [inline app/src/main/res/values/dimens.xml]: Where is it used?

File: gradlew (hunk 2 above)
Comment (Bhavnaharitsa) [reply gradlew]: sir,it showed error at that point

File: app/src/main/res/values/dimens.xml (hunk 3 above)
Comment (Bhavnaharitsa) [reply app/src/main/res/values/dimens.xml]: it is used in values-w820dp file

**Pull request closed without merging**